src/shared/fstab-util.c

```diff
@@ -105,6 +105,33 @@ static int fstab_is_same_node(const char *what_fstab, const char *path) {
         return false;
 }
 
+int fstab_has_mount_point_prefix_strv(char * const *prefixes) {
```

Comment:
frankly, i don't really see the point of constifying char** stuff... it's just a mess that requires casting everywhere. I'd drop the const. 

I know @keszybz loves these casts, but it sounds unnecessary to me.